docs/pipelines.md

```diff
@@ -0,0 +1,107 @@
+# Pipelines
+
+**monasca-docker** is home for two pipelines, one for processing **metrics** and the other one
+for processing **logs**. Each pipeline requires a set of services to be complete.
+
+## Metric pipeline
+
+**Metrics** pipeline requires following services to be launched or available
+(providing they are hosted externally):
+
+* `agent-forwarder` - launches [monasca-agent-forwarder][1]
+* `agent-collector` - launches [monasca-agent-collector][2]
+* `zookeeper` - launches [Zookeeper][3]
+* `influxdb` - launches [InfluxDB][4]
+* `kafka` - launches [Kafka][5]
+* `thresh` - launches [monasca-thresh][6]
+* `monasca-sidecar` - launches [monasca-sidecar][7]
+* `monasca-persister`- launches [monasca-persister][8]
+* `monasca` - launches [monasca-api][9]
+* `mysql` - launches [MySQL][10]
+* `keystone` - launches [Keystone][11]
+* `memcached` - launches [memcache][12]
+* `monasca-notification` - launches [monasca-notification][13]
+* `grafana` - launches [Grafana][14]
+* `cadvisor` - launched [cadvisor][15]
+
+Some of the services require initialization prior to launching, hence following
+are available:
+
+* [mysql-init][16] for `mysql`
+* [kafka-init][17] for `kafka`
+* [influxdb-init][18] for `influxdb`
+* [grafana-init][19] for `grafana`
+
+There is also:
+
+* `alarms` - that can be used to pre-create some [alarm-definitions][20] and
+  [notification-methods][21].
+
+## Log pipeline
+
+**Logs** pipeline requires following services to be launched or available
+(providing they are hosted externally):
+
+* `log-metrics` - launches [monasca-log-metrics][23]
+* `log-api` - launches [monasca-log-api][24]
+* `log-persister` - launches [monasca-log-persister][25]
+* `kibana` - launches [Kibana][26]
+* `log-transformer` - launches [monasca-log-transformer][27]
+* `elasticsearch` - launches [ElasticSearch][28]
+
+Some of the services require initialization prior to launching, hence following
+are available:
+
+* [elasticsearch-init][22] for `elasticsearch`
+* [kafka-log-init][17] for `kafka`
+
+## Dependencies
+
+Pretty much each service that is part of **monasca-docker** requires another one (or multiple)
+to run. The best way to know what is the dependency for a service is to open
```

Comment:
...way to find out what dependencies a service has, is

Reply:
Done.